active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+We recommend you avoid positioning based on message length or other UI elements set to a specific language.
+
+The notification will follow standard mirroring behavior when localized to/from languages that are right to left (RTL) or left to right (LTR). The icon will not mirror unless there is directionality. 
+
+Please view the guidance for [Adjust layout and fonts, and support RTL](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/adjust-layout-and-fonts--and-support-rtl) for more information about text localization in your notification.
+
+# Remarks
+## Usage Recommendations
+
+### When to show an information bar?
+An InfoBar should be shown when the state of the application is different from typical, expected functionality or when the user **needs** to acknowledge or potentially react to the presented information.
+
+Recommended patterns for error notifications where the application is in a negatively altered state
+  - Example: Internet connectivity is required for the application to function and is not present
+    - An error information bar should remain in view until the connection is restored if most functionality is unavailable.
+    - If some app functionality is possible, an error information bar should appear and able to be dismissed by the user.
+    - If/when the internet is restored the existing notification should update if it exists, and a new one should be created if the previous one had been dismissed. THe new/updated success-styled InfoBar should inform the user that "Internet is reconnected" so that they are aware app functionality is restored.
+  - Example: The user's subscription to your application has expired and many features are deactivated
+    - An error information bar could remain in view with a potential action button to resubscribe.
+    - Another option is to allow the user to dismiss a default-style InfoBar depending on how vital a subscription is to your application.
+
+Recommended patterns for informational notifications where the user needs to view or react to essential information
+ - Example: Terms and Conditions of the application have been updated and need to be acknowledged for continued use
+   - A default information bar could appear notifying the user the T&C have updated and link to the contents. The user acknowledges these updated T&C through dismissing the notification.
+ - Example: A virus scan that was running in the background of your application has completed
+   - A success-styled information bar could appear notifying the user the scan has completed and direct them to view the results.
+
+ Note: Users should **always** have the option to dismiss informational notifications.
+
+# API Notes
+
+### Notable Properties  
+
+| Name | Description |
+|:-:|:--|
+| IsOpen| Gets or sets a value that determines the visibility of the InfoBar. By default, is set to false. |
```

Comment:
What is the reason to have this in addition to the usual Visibility property?

Reply:
(I'm guessing, after further reading, that it's so there are events around Closing/Closed and the reason for it, etc.)